bin/build.sh

```diff
@@ -1,73 +1,81 @@
 #!/bin/bash
 
-# Makes it so that all commands are displayed as they get executed
-set -x
-
-# WCA url and absolute path to WCA logo file
 wca_url="https://www.worldcubeassociation.org/"
 wca_docs_url="https://documents.worldcubeassociation.org/"
+# Absolute paths to the logo file, the main stylesheet and the edudoc header
 logo=$(realpath "./assets/WCAlogo_notext.svg")
-# Absolute paths to the stylesheets and the edudoc header
-stylesheet=$(realpath "./assets/style.css")
-edudoc_stylesheet=$(realpath "./assets/edudoc-style.css")
+main_stylesheet=$(realpath "./assets/style.css")
 edudoc_header=$(realpath "./assets/edudoc-header.html")
 # Current date
 compile_date=$(date '+%Y-%m-%d')
 
-# Function for converting Markdown files from a given folder into PDF files using the given stylesheet
-# The first argument ($1) is the folder and the second argument is the stylesheet ($2)
+# Function for converting Markdown files from a given directory into PDF files.
+# The first argument ($1) refers to the directory name (e.g: documents or edudoc).
 convert_to_pdf() {
+  rm -rf "build/$1"
   cp -r "$1/" build/
 
-  # Find formal and legal Markdown files and build PDFs out of them.
+  # Find Markdown files and build PDFs out of them.
   find "build/$1" -name '*.md' | while read file; do
-    echo "Processing $file using the ${1} stylesheet"
+    echo "Converting ${file#build/}..."
```

Comment:
You dropped the part that prints information about the stylesheet being used. This could make debugging harder.

Reply:
You're right, and since two stylesheets can be used now, I could update this to show which stylesheets are being used.